libr/search/search.c

```diff
@@ -427,7 +348,7 @@ R_API RList *r_search_find(RSearch *s, ut64 addr, const ut8 *buf, int len) {
 
 /* --- keywords --- */
 R_API int r_search_kw_add(RSearch *s, RSearchKeyword *kw) {
-	if (!kw) return false;
+	if (!kw->keyword_length) return false;
```

Comment:
Not the place to discuss this and it was already discussed several times. You are late to the party and we spent 2 years cooking a coding style that fits well for most of us. So just follow it